.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+What have you tried to diagnose or workaround this issue?
+
+**To Reproduce**
+Steps to reproduce the behavior: (e.g. list commands or actions used to reproduce the bug)
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Detail Info**
+1) Branch name and commit hash of the 3 repositories: sof (firmware/topology), linux (kernel driver) and sof-test (test case)
+    * Kernel: {SHA}
+    * SOF: {SHA}
+    * SOF-TEST: {SHA}
+2) Test report ID (if you find it from test report)
+    * ID: {NUMBER}
+3) Test DUT Model (or a brief discribtion about the device)
+    * MODEL: {DUT MODEL}
```

Comment:
A Model may have several topologies (different in channel, with/without a component like KWD), I suggest add this information here, too.

Reply:
updated